### PR TITLE
[Maxwell]-Building-a-blog-subscription-dapp-on-Celo-part-2

### PR DESCRIPTION
**Pull request by maxzysparks**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.
- [x] I've followed all the best practices mentioned in the [here](https://docs.celo.org/community/celo-sage/complete-your-tutorial#best-practices).

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: blog/04-30-2023-Building-a-blog-subscription-dapp-on-Celo-part-2/Building-a-blog-subscription-dapp-on-Celo-part-2.md [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+  - name: Maxwell Onyeka
+    title: Technical Writer
+    url: https://github.com/maxzysparks
+    image_url: https://avatars.githubusercontent.com/u/80678178?v=4
+tags: [celo, advanced, solidity, celosage]
+hide_table_of_contents: true
+slug: /tutorials/building-a-blog-subscription-dapp-on-celo-part-2
+---
+
+![header](../../src/data-tutorials/showcase/advanced/blogg.png)
```
Comment (itxtoledo) [inline blog/04-30-2023-Building-a-blog-subscription-dapp-on-Celo-part-2/Building-a-blog-subscription-dapp-on-Celo-part-2.md:14]: You need to change the image name following other images pattern, in your case is must be "building-a-blog-subscription-dapp-on-celo-part-2.png"

Comment (maxzysparks) [reply blog/04-30-2023-Building-a-blog-subscription-dapp-on-Celo-part-2/Building-a-blog-subscription-dapp-on-Celo-part-2.md:14]: It works both ways, or was that a suggestion from Joe or Viral?

Comment (maxzysparks) [reply blog/04-30-2023-Building-a-blog-subscription-dapp-on-Celo-part-2/Building-a-blog-subscription-dapp-on-Celo-part-2.md:14]: Okay thank you, I will keep that in mind.

**Review by Ishanpathak1 — APPROVED** [review]
Approved @developerkunal

**Pull request merged**